### service to control camera focus.


int32 FocusPos


---
bool success          # True if the call succeeded
string status_message # Used to give details about success
